openbsd-games.db

```diff
@@ -1958,7 +1958,7 @@ Cover	450px-Fez_-_cover.png
 Engine	FNA
 Setup	fnaify
 Runtime	fnaify
-Store	https://www.humblebundle.com/store/fez https://polytron.itch.io/fez https://store.steampowered.com/app/224760/FEZ/
+Store	https://www.humblebundle.com/store/fez https://polytron.itch.io/fez https://store.steampowered.com/app/224760/FEZ/ https://www.gog.com/game/fez
```

Comment:
Sadly, at this point only the beta version that can be downloaded on Steam works, because the rest is too far behind FNA. With this, I think this should maybe rather be reduced to just the Steam and Humble links (because Humble purchase should come with a Steam key).